### PR TITLE
#271 - Added skip_update param reading to RpmUpload.Request

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #271 
Added skip_update param reading to `RpmUpload.Request` along with tests

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#279](https://github.com/artipie/rpm-adapter/issues/279) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/395dad4a-97a0-4db0-94a9-c67e1dcbe6fc); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/rpm/http/RpmUpload.java [hunk 1]
```diff
@@ -146,12 +147,16 @@ public Key file() {
          * @return Override param value, <code>false</code> - if absent
          */
         public boolean override() {
-            return Optional.ofNullable(new RequestLineFrom(this.line).uri().getQuery())
-                .map(
-                    query -> Streams.stream(Splitter.on("&").split(query))
-                        .anyMatch(part -> part.equals("override=true"))
-                )
-                .orElse(false);
+            return this.params().anyMatch(part -> part.equals("override=true"));
```
Comment (Vatavuk) [inline src/main/java/com/artipie/rpm/http/RpmUpload.java:150]: @olegmoz `part -> part.equals("...")` can be extracted to a private method to make code more readable and avoid code duplication. We can have than `this.params().anyMatch(equalTo("override=true")) `

Comment (olegmoz) [reply src/main/java/com/artipie/rpm/http/RpmUpload.java]: @Vatavuk agreed, please review the update

**Comment by olegmoz** [comment]
@Vatavuk fixed JavaDoc, please re-approve

**Comment by 0crat** [comment]
Job was finished in 1 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will find at least 3 issues during next CR, thanks!

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@Vatavuk quality acceptable

**Comment by Vatavuk** [comment]
@g4s8 please remove this issue from scope

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**